reference/docs-conceptual/learn/ps101/02-help-system.md

```diff
@@ -0,0 +1,666 @@
+    articles that include that word in their contents.
+
+    The names of conceptual articles, such as about_Objects, must be entered in English, even in
+    non-English versions of PowerShell.
+
+    Required?                    false
+    Position?                    0
+    Default value                None
+    Accept pipeline input?       True (ByPropertyName)
+    Accept wildcard characters?  true
+```
+
+Notice that in the previous example, the **Parameter** parameter was used with the Help function to
+only return information from the help topic for the **Name** parameter. This is much more concise
+than trying to manually sift through what sometimes seems like a hundred page help topic.
+
+Based on those results, you can see that the **Name** parameter is positional and must be specified
+in position zero (the first position) when used positionally. The order that parameters are
+specified in doesn't matter if the parameter name is specified.
+
+One other important piece of information is that the **Name** parameter expects the datatype for its
+value to be a single string, which is denoted by `<String>`. If it accepted multiple strings, the
+datatype would be listed as `<String[]>`.
+
+Sometimes you simply don't want to display the entire help topic for a command. There are a number
+of other parameters besides Full that can be specified with `Get-Help` or Help. Try running the
+following commands on your Windows 10 lab environment computer:
+
+- Get-Help -Name Get-Command -Full
+- Get-Help -Name Get-Command -Detailed
+- Get-Help -Name Get-Command -Examples
+- Get-Help -Name Get-Command -Online
+- Get-Help -Name Get-Command -Parameter Noun
+- Get-Help -Name Get-Command -ShowWindow
+
+I typically use help `<command name>` with the **Full** or **Online** parameter. If I'm only
+interested in the examples, I'll use the **Examples** parameter and if I'm only interested in a
+specific parameter, I'll use the Parameter parameter. The **ShowWindow** parameter opens the help
+topic in a separate searchable window that can be placed on a different monitor if you have multiple
+monitors. I've avoided the **ShowWindow** parameter because there's a known bug where it doesn't
+display the entire help topic.
+
+If you want help in a separate window, my recommendation is to either use the **Online** parameter
+or use the **Full** parameter and pipe the results to `Out-GridView`, as shown in the following
+example.
+
+```powershell
+help Get-Command -Full | Out-GridView
+```
+
+Both the `Out-GridView` cmdlet and the **ShowWindow** parameter of the `Get-Help` cmdlet require an
+operating system with a GUI (Graphical User Interface). They will generate an error message if you
+attempt to use either of them on Windows Server that's been installed using the server core (no-GUI)
+installation option.
+
+To use `Get-Help` to find commands, use the asterisk (\*) wildcard character with the **Name**
```

Comment:
That asterisk should be codefenced instead of escaped to match other articles.

Reply:
Fixed